### PR TITLE
Add Cloud Feeds API docs

### PR DESCRIPTION
**Pull request by meker12**

### CODE REVIEW

**Comment by meker12** [comment]
@ktbartholomew or @smashwilson  Please review and merge (hopefully).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-7b39fe9442/docs/)

File: index.rst [hunk 1]
```diff
@@ -527,6 +527,26 @@
                <div class="card purple">
                    <div class="card-content">
                        <div class="card-title">
+                           <h4 id="docs-cloud-keep">Cloud Feeds</h4>
+                       </div>
+                       <div class="card-body">
+                           <div class="list">
+                               <div class="list-column">
+                                   <ul>
+                                       <li><a href="docs/cloud-feeds/v1/developer-guide/#document-getting-started">Getting Started</a></li>
```
Comment (smashwilson) [inline index.rst]: You're missing a leading `/` here:

```
<a href="/docs/cloud-feeds/v1/...
```

File: index.rst [hunk 2]
```diff
@@ -527,6 +527,26 @@
...
+                                       <li><a href="/docs/cloud-feed/v1/developer-guide/#document-api-reference">API Reference</a></li>
```
Comment (smashwilson) [inline index.rst]: Also, this link should be `/docs/cloud-feeds/`, not `/docs/cloud-feed/`. :sparkles:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-9d08a10370/docs/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-9d08a10370/docs/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-565d69be61/docs/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-8436df1c7f/docs/)

**Pull request merged**